src/index.js

```diff
@@ -0,0 +1,159 @@
+export default class toColor {
+  HUE_MAX = 360;
+
+  constructor(seed, options) {
+    this.options = options || {};
+    this.seed = typeof seed === 'string' ? this._stringToInteger(seed) : seed;
+    this.known = [];
+  }
+
+  getColor() {
+    const { distance, knownMax } = this.options;
+    const h = this._pickHue();
+    const s = this._pickSaturation(h);
+    const b = this._pickBrightness(h, s);
+    const hsl = this._HSVtoHSL(h, s, b);
+    const PASSABLE_DISTANCE = typeof distance !== 'undefined' ? distance : 37; 
```

Comment:
Consider hard coding `37` and some of your other magic numbers to named constants (DEFAULT_PASSABLE_DISTANCE).